docs/api/scheduler.md

```diff
@@ -85,6 +85,12 @@ Displays general information about the applications like used resources, queue n
 
 **Method** : `GET`
 
+**Query Params** : 
+
+1. queue=<fully qualified queue name\>
+
+Fully qualified queue name filters the applications based on exact match. Passing only leaf queue name doesn't give the expected behaviour. For example, /ws/v1/apps?queue=root.default return the applications running in "root.default" queue.
```

Comment:
> Passing only leaf queue name doesn't give the expected behaviour.

Not completely sure what does this mean. Can you pls refine the text.

Reply:
Can we say "Fully qualified queue name filters the applications based on exact match. For example, /ws/v1/apps?queue=root.default return the applications running in "root.default" queue. Passing only leaf queue name ("default") doesn't return the expected results." ?

Reply:
Ah I see. You want to emphasize the parameter should be using the fully qualified name. How about:

> The fully qualified queue name to filter the applications that run within the given queue. For example, /ws/v1/apps?queue=root.default return the applications running in "root.default" queue.

Reply:
Looks good.